File: JS-Basics/Conditional Statements Advanced/Exercise/04.js
function fishingTrip(budget, season, countFishman) {
    let priceShip = 0;
    let discount = 0;

    if (season === "Spring") {
        priceShip = 3000;
    } else if (season === "Summer" || season === "Autumn") {
        priceShip = 4200;
    } else if (season === "Winter") {
        priceShip = 2600;
    }

    if (countFishman <= 6) {
        discount += 0.1;
    } else if (countFishman >= 7 && countFishman <= 11) {
        discount += 0.15;
    } else if (countFishman >= 12) {
        discount += 0.25;
    }

    let total = priceShip - (priceShip * discount);

    if (season !== "Autumn" && countFishman % 2 === 0) {
        total *= 0.95;
    }

    let leftMoney = Math.abs(budget - total).toFixed(2);


    if (budget >= total) {
        console.log(`Yes! You have ${leftMoney} leva left.`);
    } else {
        console.log(`Not enough money! You need ${leftMoney} leva.`);
    }
}

File: Basics/Conditional Statements Advanced/Exercise/04.py
budget = int(input())
season = input()
count_fishman = int(input())

price_ship = 0
discount = 0

if season == "Spring":
    price_ship = 3000

    if count_fishman <= 6:
        discount += 0.1
    elif 7 <= count_fishman <= 11:
        discount += 0.15
    elif count_fishman >= 12:
        discount += 0.25

elif season == "Summer":
    price_ship = 4200

    if count_fishman <= 6:
        discount += 0.1
    elif 7 <= count_fishman <= 11:
        discount += 0.15
    elif count_fishman >= 12:
        discount += 0.25

elif season == "Autumn":
    price_ship = 4200

    if count_fishman <= 6:
        discount += 0.1
    elif 7 <= count_fishman <= 11:
        discount += 0.15
    elif count_fishman >= 12:
        discount += 0.25

elif season == "Winter":
    price_ship = 2600

    if count_fishman <= 6:
        discount += 0.1
    elif 7 <= count_fishman <= 11:
        discount += 0.15
    elif count_fishman >= 12:
        discount += 0.25

total = price_ship - (price_ship * discount)

if season != "Autumn" and count_fishman % 2 == 0:
    total = total * 0.95

left_money = abs(budget - total)

if budget >= total:
    print(f"Yes! You have {left_money:.2f} leva left.")
else:
    print(f"Not enough money! You need {left_money:.2f} leva.")
